docs/services.rst

```diff
@@ -111,6 +115,11 @@ is ``'Caller-Destination-Number'``) and ``kwargs``.
 The ``pattern`` must be matched against the ``field`` *event header* in order for
 the *route* to be called with ``kwargs`` (i.e. ``reject_international(**kwargs)``).
 
+
+You can run this app directly using ``switchy serve``::
+
+    switchy serve freeswitch1.net freeswitch2.net --app ./dialplan.py:router
+
```

Comment:
should be `./dialplan.py:Router`, right ? Which mean class `Router` in `dialplan.py`.